spec/acceptance/suites/offline/00_default_spec.rb

```diff
@@ -0,0 +1,158 @@
+require 'spec_helper_acceptance'
+
+describe 'Offline mode' do
+  hosts.each do |host|
+    context "on #{host}" do
+      let(:vagrant_rpm) { 'https://releases.hashicorp.com/vagrant/2.2.3/vagrant_2.2.3_x86_64.rpm' }
+      let(:build_user) { 'build_user' }
+      let(:build_user_cmd) { "runuser #{build_user} -l -c" }
+
+      # Not sure if this is a QEMU thing with the image or something else
+      it 'works around a CentOS curl bug with libvirt' do
+        on(host, %(touch /etc/sysconfig/64bit_strstr_via_64bit_strstr_sse2_unaligned))
```

Comment:
(•_•)  Is there a URL about this bug that we can reference here?

Reply:
Uh...sort of? https://access.redhat.com/articles/2050743

It was a joy of a day with `strace` that let me actually figure it out so the bug looks only loosely relevant.